src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -215,6 +220,7 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
                             "end": release.date
                             + timedelta(threshold.window_in_seconds),  # start + threshold.window
                             "release": release.version,
+                            "project_slug": project.slug,
                             "project_id": project.id,
```

Comment:
is it redundant to serialize the project in `SerializedThreshold` and then also include `project_slug` and `project_id` in `EnrichedThreshold`?

Reply:
Yea - it's a little bit redundant.
BUT - the full serialized project is just really noisy so figured i'd bring it up to surface level 😬 
I could also bring up the environment name i suppose....

Reply:
in that case is it even useful to serialize the project in SerializedThreshold?

Reply:
Ya - figured it may still be useful for customers to see the fully enriched data